WordPress/Sniffs/XSS/EscapeOutputSniff.php

```diff
@@ -441,4 +440,18 @@ protected function mergeFunctionLists() {
 		}
 	}
 
+	/**
+	 * Checks whether short_open_tag is enabled.
+	 *
+	 * Since PHP 5.4, <?= is always available.
+	 *
+	 * @return bool False if short_open_tag is not enabled, true otherwise
+	 */
+	public function is_short_open_tag_enabled() {
```

Comment:
The returned value of this function is not going to change during the PHPCS run, so I'd like to suggest adding a class (private) property instead, assigning the value to the property at the start of the `register()` function and referring to the property later on for efficiency.